en/config_rover/mecanum.md

```diff
@@ -0,0 +1,298 @@
+:::
+
+## Acro Mode
+
+To set up [Acro mode](../flight_modes_rover/mecanum.md#acro-mode) navigate to [Parameters](../advanced_config/parameters.md) in QGroundControl and set the following parameters:
+
+1. [RM_WHEEL_TRACK](#RM_WHEEL_TRACK) [m]: Measure the distance from the centre of the right wheel to the centre of the left wheel.
+
+   ![Wheel track](../../assets/airframes/rover/rover_differential/wheel_track.png)
+
+2. [RM_MAX_YAW_RATE](#RM_MAX_YAW_RATE) [deg/s]: This is the maximum yaw rate you want to allow for your rover.
+   This will define the stick-to-yaw-rate mapping for all manual modes using closed loop yaw control and set an upper limit for the yaw rate setpoint for all [auto modes](#auto-modes).
+3. [RM_MAX_THR_YAW_R](#RM_MAX_YAW_RATE) [m/s]: This parameter is used to calculate the feed-forward term of the closed loop yaw rate control.
+   The controller calculates the required speed difference between the left and right motor to achieve the desired yaw rate.
+   This desired speed difference is then linearly mapped to normalized motor commands.
+   To get a good starting value for this parameter drive the rover in manual mode forwards at full throttle and note the ground speed of the vehicle.
+   Then enter _twice_ this value for the parameter.
+   <a id="RM_YAW_RATE_P_TUNING"></a>
+
+   ::: tip
+   To further tune this parameter, first make sure you set [RM_YAW_RATE_P](#RM_YAW_RATE_P) and [RM_YAW_RATE_I](#RM_YAW_RATE_I) to zero.
+   This way the yaw rate is only controlled by the feed-forward term, which makes it easier to tune.
+   Now put the rover in [Acro mode](../flight_modes_rover/mecanum.md#acro-mode) and then move the right-stick of your controller to the right and/or left and hold it at a few different levels for a couple of seconds each.
+   Disarm the rover and from the flight log plot the _yaw_rate_setpoint_ and actual _yaw_rate_ from the [RoverMecanumSetpoint](../msg_docs/RoverMecanumStatus.md) over each other.
+   If the actual yaw rate of the rover is higher than the yaw rate setpoint, increase [RM_MAX_THR_YAW_R](#RM_MAX_YAW_RATE).
+   If it is the other way around decrease the parameter and repeat until you are satisfied with the setpoint tracking.
+   :::
+
+4. [RM_YAW_RATE_P](#RM_YAW_RATE_P) [-]: Proportional gain of the closed loop yaw rate controller.
+   Unlike the feed-forward part of the controller, the closed loop yaw rate control will compare the yaw rate setpoint with the measured yaw rate and adapt to motor commands based on the error between them.
+   The proportional gain is multiplied with this error and that value is added to the motor command.
+   This way disturbances like uneven grounds or external forces can be compensated.
+
+   ::: tip
+   This parameter can be tuned the same way as [RM_MAX_THR_YAW_R](#RM_YAW_RATE_P_TUNING).
+   If you tuned [RM_MAX_THR_YAW_R](#RM_MAX_YAW_RATE) well, you might only need a very small value.
```

Comment:
I don't get " This parameter can be tuned the same way as [RM_MAX_THR_YAW_R]"

That says:

> To get a good starting value for this parameter drive the rover in manual mode forwards at full throttle and note the ground speed of the vehicle. Then enter twice this value for the parameter.

So the value is 2 times ground speed. 
So why would that be small and why affected by tuning for `RM_MAX_THR_YAW_R`?

Reply:
Updated this in https://github.com/PX4/PX4-user_guide/pull/3411/commits/ab30f0666db49e0a694bfe92f2f89ca2f9760945